### PR TITLE
Add typing to the library

### PR DESCRIPTION
**Pull request by PabloSzx**
Thanks for this package, is very useful.
But i needed the typescript types so i made them and here it is, useful for everyone, even for no typescript users, since it helps the editors for autocomplete.

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import * as React from "react";
```
Comment (Norserium) [inline index.d.ts]: I prefer `import {ReactNode, CSSProperties} from 'react'` here.

Comment (PabloSzx) [reply index.d.ts]: Me too, but usually when using .d.ts definitions all i see is "import * as React ", i guess it should matter, changes made

File: index.d.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import { CSSProperties, FC, ReactNode } from "react";
```
Comment (Norserium) [inline index.d.ts]: I would prefer to move this file to `/types/index.d.ts` and change `package.json` accordingly.

**Comment by Norserium** [comment]
Thank you very much for your contribution, it's the important thing that I always forget to do. I reviewed your code, please see my remarks.

**Comment by Norserium** [comment]
It look pretty good. I'm going to merge this pull request.

**Comment by Norserium** [comment]
I almost forgot, could you change the package version to `1.4.3`? I will publish the package too.

**Comment by Norserium** [comment]
Thanks again! I've merged the pull request. Now I will publish your changes.

**Comment by Norserium** [comment]
Published.

**Pull request merged**